application/models/user.go

```diff
@@ -434,6 +434,7 @@ func (u *User) GetPhotoURL() (*string, error) {
 
 // Save wraps DB.Save() call to check for errors and operate on attached object
 func (u *User) Save() error {
+	u.Nickname = domain.RemoveUnwantedChars(u.Nickname, "-_ .,'&@")
```

Comment:
Should we ensure the resulting string has at least one visible character in it? (Would invisible characters just be spaces, and zero-width joiners?)

Reply:
Good thought. I just checked and it is handled by the User validation. The UI gives the "Something went wrong We had a problem updating the user profile." message and "nickname: Nickname can not be blank." is logged.

Reply:
I somehow ignored the word "visible" in your message. I'll take another look at this.

Reply:
Done